### DISCUSSION TITLE
Show a subform inside a form

### DISCUSSION THREAD

**Discussion by Webmasters-WPNGO** [wintercms/winter#1008] [Q&A]
I'm using Builder. I have a non-standard task. I want the user to select options using a dropdown and after selecting, other forms with a different model will be shown.

Is it really possible to do this using Builder?

**Comment by LukeTowers** [comment]
Probably not with the builder, could you explain more about what you're trying to accomplish, using the actual terminology instead of any attempts at abstracting it if possible?

**Comment by Webmasters-WPNGO** [comment]
My plugin creates a landing page with the user choosing a template when creating a post. When creating a record, it is more convenient to select a form template.

Maybe I'm wrong and I'm complicating things. Maybe there is a simpler solution.

**Reply by LukeTowers** [reply]
Why not use the Static Pages plugin with different layouts?

**Comment by Webmasters-WPNGO** [comment]
The simplest solution I see is to enable the visibility of fields when selecting an option in the dropdown.

**Comment by Webmasters-WPNGO** [comment]
> Why not use the Static Pages plugin with different layouts?

We need a lot of custom fields, I don’t think this can be solved within Static Pages. With the help of triggers for fields, it is still possible to solve the problem.

Thanks for your support!

**Reply by LukeTowers** [reply]
@WebProviderNGO you might be surprised. With the [Winter.Blocks](https://github.com/wintercms/wn-blocks-plugin) plugin and [custom page fields](https://github.com/wintercms/wn-pages-plugin/blob/main/docs/documentation.md#custom-page-fields) / [dynamic syntax parser](https://wintercms.com/docs/v1.2/docs/services/parser#dynamic-syntax-parser) you can accomplish a lot.

**Reply by Webmasters-WPNGO** [reply]
But this is very interesting! :) Thanks, I'll look into it.

**Comment by mjauvin** [comment]
If it's for the backend, nestedforms with triggers work very well for this scenario.

**Reply by Webmasters-WPNGO** [reply]
Thank you, yes, this is the first method I used. Then I began to think whether it was possible to call another within one form.

**Comment by mjauvin** [comment]
> Thank you, yes, this is the first method I used. Then I began to think whether it was possible to call another within one form.

Yes, you can nest many levels as far as I know.

**Reply by Webmasters-WPNGO** [reply]
This is true. But it looks hard and the user starts to get angry :)
